opslevel/resource_opslevel_service.go

```diff
@@ -246,7 +254,7 @@ func resourceServiceRead(d *schema.ResourceData, client *opslevel.Client) error
 	if err := d.Set("tier_alias", resource.Tier.Alias); err != nil {
 		return err
 	}
-	if err := d.Set("owner_alias", resource.Owner.Alias); err != nil {
+	if err := d.Set("owner", resource.Owner.Alias); err != nil {
```

Comment:
we need to make sure both fields are set still since we are not removing this field only deprecating it.